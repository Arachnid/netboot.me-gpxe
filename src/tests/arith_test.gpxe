#!gpxe
echo $(17*2/(12-3))							
echo $(32 == 4*(10-2)) $(12 >= 3) $(12 < 3)		
echo $(y == y)								
echo $(-(-42) )	
echo $(4-+2*4*(5-2))
echo $(y == -2)
set a $(2*5)
echo $(${a} << 4 % 256)
echo $(12*((3+4)/2))
set i 258
echo $(12*3/2) $(45*2/(44-2))
set num 10
set serv ${num}.0.0.$(${i} & 255)
echo ${serv}
