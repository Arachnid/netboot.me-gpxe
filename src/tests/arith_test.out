3
1 1 0
1
42
-20
0
160
36
18 2
10.0.0.2
No more network devices

Press Ctrl-B for the gPXE command line...                                         
